app/forms/sections/items/ItemPackagingSealTypeFormProvider.scala

```diff
@@ -53,11 +54,10 @@ object ItemPackagingSealTypeFormProvider {
   val sealTypeInvalidErrorKey = "itemPackagingSealType.error.sealType.invalid"
   val sealTypeLengthErrorKey = "itemPackagingSealType.error.sealType.length"
 
-  val answerRequiredErrorKey = "itemPackagingSealType.error.required"
-  val answerInvalidErrorKey = "itemPackagingSealType.error.invalid"
-  val answerLengthErrorKey = "itemPackagingSealType.error.length"
+  val sealInformationInvalidErrorKey = "itemPackagingSealType.error.invalid"
+  val SealTypeInformationLengthErrorKey = "itemPackagingSealType.error.length"
```

Comment:
Change to `sealTypeInformationLengthErrorKey`

Reply:
updating